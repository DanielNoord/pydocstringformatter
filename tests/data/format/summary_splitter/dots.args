--max-summary-lines=2
