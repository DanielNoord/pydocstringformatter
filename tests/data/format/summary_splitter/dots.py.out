class MyClass:
    """We shouldn't split sys.path."""


def func():
    """We should not add extra lines after the dot
    to it.
    """


def func():
    """We should not split for e.g. here."""


def func():
    """We should not split for i.e. here."""


def func():
    """We should not add line after i.e., sys.path, e.g., etc. but etc. in particular.

    is harder right ?
    """
